build.sbt

```diff
@@ -123,7 +123,8 @@ lazy val common = (project in file("modules/common"))
   .settings(automateHeaderSettings(IntegrationTest))
   .settings(
     libraryDependencies ++= Seq(
-      "com.amazonaws" % "aws-java-sdk-core" % awsSdkVersion % Optional
+      "software.amazon.awssdk" % "auth" % awsSdkVersion,
+      "software.amazon.awssdk" % "sdk-core" % awsSdkVersion % Optional
```

Comment:
Oh wow, TIL: https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html.

I had never bumped into this concept of optional dependency 🤦‍♂️